lib/travis/api/v3/queries/subscriptions.rb

```diff
@@ -7,7 +7,29 @@ def all(user_id)
 
     def create(user_id)
       client = BillingClient.new(user_id)
-      client.create_subscription(params)
+      client.create_subscription(
+        :plan => params['plan'],
+        :coupon => params['coupon'],
+        :organization_id => params['organization_id'],
+        :billing_info => {
+          :first_name => params['billing_info.first_name'],
```

Comment:
If you declare the params in here in a [certain way](https://github.com/travis-ci/travis-api/commit/fba9a87c3956640fc49f520daba28984f732cde7), you can access them as hashes which will be a lot shorter. e.g.

```ruby
params :foo, :bar, :baz, prefix: :billing_info
params :qux, :quux, prefix: :credit_card_info

billing_info_params
  # => { 'foo' => 'a', 'bar' => 'b', 'baz' => 'c' }

credit_card_info_params
  # => { 'qux' => 'x', 'quux' => 'y' }
```

Reply:
Thank you Joe!! That makes it easier